presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -268,6 +269,23 @@ public void testTimestamp()
         // testing this is hard because of https://github.com/prestodb/presto/issues/7122
     }
 
+    @Test
+    public void testJson()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
+        jdbcSqlExecutor.execute("CREATE TABLE tpch.test_json_data_type(key varchar(5), json_column json, jsonb_column jsonb)");
+        assertUpdate("INSERT INTO tpch.test_json_data_type VALUES ('1', json'{\"x\":123}',json'{\"x\": 123}' )", 1);
+        assertQuery("SELECT * FROM tpch.test_json_data_type", "SELECT '1' \"key\", '{\"x\":   123}' json_column, '{\"x\":123}' jsonb_column");
+        try {
```

Comment:
`try` should encompass `INSERT` and `SELECT` above

Reply:
done